config/data/langs.toml

```diff
@@ -1016,6 +1016,7 @@ Sys.argv
 
 [Odin]
 args       = [ '/usr/bin/odinwrapper', '-' ]
+env        = [ 'PATH=/usr/local/bin:/usr/bin:/bin:$PATH' ]
```

Comment:
I don't think the $PATH at the end will be expanded as this doesn't get run by a shell. TBH I doubt it's needed as those three paths cover the major three so just drop the `:$PATH`

Reply:
> I don't think the $PATH at the end will be expanded as this doesn't get run by a shell. TBH I doubt it's needed as those three paths cover the major three so just drop the `:$PATH`

Correct, just good habit.